nservicebus/upgrades/5to6.md

```diff
@@ -166,6 +166,19 @@ We've separated the callback support from the NServiceBus core. This means that
 
 The differences in the API are fully covered in [handling responses on the client side](/nservicebus/messaging/handling-responses-on-the-client-side.md).
 
+## Message handlers
+
+The handler method on `IHandleMessages<T>` now returns a Task. In order to convert your synchronous code add `return Task.FromResult(0);` to the end of your handler methods. In order to leverage async code, add the async keyword to your handler method and use `await` for async methods.
```

Comment:
more people should be using async. web call, disk io, db calls. so i think have the async version first